payjoin/src/receive/mod.rs

```diff
@@ -543,6 +543,24 @@ impl ProvisionalProposal {
             input.bip32_derivation = BTreeMap::new();
             input.partial_sigs = BTreeMap::new();
         }
+        let sender_input_indexes = self.get_sender_input_indexes();
+        for i in sender_input_indexes {
+            log::trace!("Clearing sender input {}", i);
+            self.payjoin_psbt.inputs[i].non_witness_utxo = None;
+            self.payjoin_psbt.inputs[i].witness_utxo = None;
+            self.payjoin_psbt.inputs[i].final_script_sig = None;
+            self.payjoin_psbt.inputs[i].final_script_witness = None;
+            self.payjoin_psbt.inputs[i].tap_key_sig = None;
+        }
+
+        Ok(PayjoinProposal {
+            payjoin_psbt: self.payjoin_psbt,
+            owned_vouts: self.owned_vouts,
+            params: self.params,
+        })
+    }
+
+    fn get_sender_input_indexes(&self) -> Vec<usize> {
```

Comment:
The `get_` prefix is typically not used by rust functions [by convention](https://rust-lang.github.io/api-guidelines/naming.html#getter-names-follow-rust-convention-c-getter). Especially where it's not a getter (returning a field from a struct) in the typical sense here.